tests/unit/test_snap.py

```diff
@@ -290,6 +290,37 @@ def test_can_run_snap_commands(self, mock_subprocess):
             ["snap", "install", "foo", "--classic", '--revision="123"'], universal_newlines=True
         )
 
+    @patch("charms.operator_libs_linux.v2.snap.subprocess.check_output")
+    def test_can_run_snap_commands_devmode(self, mock_subprocess):
```

Comment:
I think it'd be better if this parameter were called `mock_check_output` as that's the function we're mocking, not the `subprocess` module itself.

Reply:
Done